tools/test-runner-cli.ts

```diff
@@ -560,8 +560,11 @@ function run(config: Test.Config) {
     logger.info('TestRunnerCli.Run', '(4/4) Running karma to start test runner...');
     const karmaCommand = path.join(npmBin, 'karma');
     // currently only ChromeDebug, ChromeTest, Edge, Firefox and Electron browsers are supported
-    const browser = (env === 'chrome') ? (debug ? 'ChromeDebug' : 'ChromeTest') :
-                                         (env === 'edge') ? 'Edge' : (env === 'firefox') ? 'Firefox' : 'Electron';
+    const browser = (env === 'chrome') ?
```

Comment:
maybe this is a good time to use a `if` or `switch` statement to replace the super long expression